src/cf-buttons/src/atoms/buttons-with-icons.less

```diff
@@ -1,28 +1,22 @@
 @import '../atoms/buttons.less';
 
-// Icon directions
+// Icon locations
 
-.a-btn_icon-position-left {
-  &:before {
+.a-btn_icon-position-left:before {
     padding-right: unit( 10.5px / @btn-font-size, em );
     border-right: 1px solid mix( @btn-bg, @btn-text, 50% );
     margin-right: unit( 7px / @btn-font-size, em );
-  }
 }
 
-.a-btn_icon-position-right {
-  &:after  {
+.a-btn_icon-position-right:after  {
```

Comment:
Discussed while pairing, but for the record: renaming these to be a modifier, now that modifiers are not needed at the choose-an-icon level (see removed code below).

We settled on `.a-btn__icon-on-left` and `.a-btn__icon-on-right`.